spec/features/markup_helper_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+include MarkupHelper
+
+# Scenario: Tokens are properly interpolated into schedule lists
+#   verify that @info gets interpolated
+#   verify that [text](link) gets interpolated
+#   verify that when the link markup is  == [Divider](url), then an image url interpolated.
+
+describe MarkupHelper do
```

Comment:
Good to see this. Was going to note that there's no way I'd want to ship that parser without unit tests.